pagecarton/core/library/Ayoola/Application.php

```diff
@@ -1382,82 +1365,98 @@ function_exists( 'http_response_code' ) ? http_response_code(404) : null;
 					}
 				}
 			}
-	//	var_export( $PAGE_INCLUDE_FILE );     
-	//	var_export( $PAGE_TEMPLATE_FILE );     
 		}
 		while( false );
-	//	var_export( $PAGE_INCLUDE_FILE );     
-	//	var_export( $PAGE_TEMPLATE_FILE );     
-	//	var_export( $pagePaths['template'] );     
-	//	var_export( Ayoola_Loader::checkFile( $pagePaths['template'] ) );     
+	//	var_export( $PAGE_INCLUDE_FILE );
+	//	var_export( $PAGE_TEMPLATE_FILE );
+	//	var_export( $pagePaths['template'] );
+	//	var_export( Ayoola_Loader::checkFile( $pagePaths['template'] ) );
+		//	The normal page
+		try
+		{
+			//	Check if page options permits
+/* 			$pageOptions = Ayoola_Page::getCurrentPageInfo();
+			@$pageOptions = $pageOptions['page_options'] ? : array();
+			$access = new Ayoola_Access();
+			if(	( in_array( 'logged_in_hide', $pageOptions )  && $access->isLoggedIn() )
+			|| 	( in_array( 'logged_out_hide', $pageOptions ) && ! $access->isLoggedIn() )
+			)
+			{
+				self::view();
+				exit();
+			}
+ */		}
+		catch( Ayoola_Exception $e ){ self::view(); exit(); }
 
-	
-		//	Put in Access Restriction
-		$noRestriction ? : self::restrictAccess();
+	//	$table = new Application_User_CloudCopy();
+	//	var_export( file_get_contents( "php://input" ) );
+	//	$table->select();
 
+		//	Put in Access Restriction
+		self::restrictAccess();

```

Comment:
This was there because of a special case where we don't need restriction. This will break sites that make use of "Theme Pages" that should always be authorized unless their authorization parameters are specifically set.